opm/simulators/utils/readDeck.cpp

```diff
@@ -248,21 +253,19 @@ void readDeck(int rank, std::string& deckFilename, std::unique_ptr<Opm::Deck>& d
     {
         Opm::eclStateBroadcast(*eclipseState, *schedule, *summaryConfig);
     }
-    catch(const std::exception& e)
+    catch(const std::exception& broadcast_error)
     {
-        failureMessage = e.what();
+        failureMessage = broadcast_error.what();
         parseSuccess = 0;
     }
 
 #endif
 
     if (*errorGuard) { // errors encountered
         parseSuccess = 0;
+        errorGuard->clear();
     }
 
-    // print errors and warnings!
-    errorGuard->dump();
```

Comment:
Are these guaranteed to be printed elsewhere?

Reply:
As mentioned - there is no hard guarantee no; but the intention is that the `ParseContext::handleError()` will print these messages - so with the current implementation - yes we are guaranteed that the messages has been printed already.